testing/matrix-sdk-integration-testing/src/helpers.rs

```diff
@@ -84,8 +84,8 @@ impl TestClientBuilder {
         let mut client_builder = Client::builder()
             .user_agent("matrix-sdk-integration-tests")
             .homeserver_url(homeserver_url)
-            // Disable Simplified MSC3575 for the integration tests as, at the time of writing
-            // (2024-07-15), we use a Synapse version that doesn't support Simplified MSC3575.
+            // Disable MSC4186 for the integration tests as, at the time of writing
+            // (2024-07-15), we use a Synapse version that doesn't support MSC4186.
```

Comment:
(Not for this PR, but) we should try to move forward on this maybe?

Reply:
For sure, but not today (maybe tomorrow?).

Reply:
For the record, it's already tracked here https://github.com/matrix-org/matrix-rust-sdk/issues/3698